### PR TITLE
implement uninstall verb

### PR DESCRIPTION
**Pull request by dirk-thomas**
Related to ament/ament_cmake#13

### CODE REVIEW

**Comment by esteve** [comment]
+1

File: ament_tools/verbs/uninstall/cli.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# Copyright 2014 Open Source Robotics Foundation, Inc.
```
Comment (wjwwood) [inline ament_tools/verbs/uninstall/cli.py]: 2015?

**Comment by wjwwood** [comment]
Probably need to mention both `on_test` and `on_uninstall` in the docs for the base class:

https://github.com/ament/ament_tools/blob/master/ament_tools/build_type.py#L95

Otherwise +1

File: ament_tools/verbs/uninstall/cli.py (hunk 1 above)
Comment (dirk-thomas) [inline ament_tools/verbs/uninstall/cli.py]: Fixed.

**Comment by dirk-thomas** [comment]
I updated `BuildType` to also implement the `on_test` and `on_uninstall` methods with `raise NotImplementedError`.

**Comment by dirk-thomas** [comment]
I will go ahead and merge this. If anybody has more comment please feel free to add them afterwards.

**Pull request merged**